calico/felix/endpoint.py

```diff
@@ -233,6 +395,7 @@ def __init__(self, config, combined_id, ip_type, iptables_updater,
         self._chains_programmed = False
         self._iptables_in_sync = False
         self._device_in_sync = False
+        self._profile_ids_dirty = False
```

Comment:
I couldn't think of one.  The change from profile to policy was a last-minute one; I'd rather leave "profile" as the generic term in Felix rather than churn some more.